packages/cli/src/util/validate-config.ts

```diff
@@ -93,6 +93,28 @@ const imagesSchema = {
   },
 };
 
+const cronsSchema = {
+  type: 'array',
+  minItems: 0,
+  items: {
+    type: 'object',
+    additionalProperties: false,
+    required: ['path', 'schedule'],
+    properties: {
+      path: {
+        type: 'string',
+        minLength: 1,
+        maxLength: 512,
+      },
+      schedule: {
+        type: 'string',
+        minLength: 1,
+        maxLength: 256,
+      },
+    },
+  },
+};
```

Comment:
Is there a test for this?

Reply:
Will add one 👍

Reply:
Lets add multiple:

- path without schedule
- path and length but exceeding 512
- path and length but empty strings

Reply:
Added tests here: 12f076fb97087bbd01fce130365c29a59d838e24